testing/demo/pkg/setup/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+	err := InitializeGroth16LightClientOnSimapp()
```

Comment:
[no change needed] FYI for other reviewers, this command also creates the channel

Reply:
yeah I wasn't sure how to add that as part of the signature as well. I'll add a godoc actually. can be done in a diff pr.